ydb/docs/en/core/yql/reference/syntax/alter-user.md

```diff
@@ -1,17 +1,75 @@
 # ALTER USER
 
-Changes the user password.
+Alters a database user.
 
 ## Syntax
 
 ```yql
 ALTER USER user_name [ WITH ] option [ ... ]
 ```
 
-* `user_name`: The name of the user.
-* `option` — The command option:
+{% include [!](../../../_includes/user-options.md) %}
 
-  * `PASSWORD 'password'` — changes the password to `password`.
-  * `PASSWORD NULL` — sets an empty password.
-  * `NOLOGIN` - disallows user login (user lockout).
-  * `LOGIN` - allows user login (user unlocking).
+{% include [!](../../../_includes/do-not-create-users-in-ldap.md) %}
+
+## PASSWORD
+
+Users can change only their own passwords. Database administrators can change a password of any user.
```

Comment:
What's a database administrator in this context? It needs to refer to the proper place in general security documentation.